package.json

```diff
@@ -52,7 +52,8 @@
   "resolutions": {
     "ansi-regex": "^5.0.1",
     "nth-check": "^2.0.1",
-    "follow-redirects": "^1.14.7"
+    "follow-redirects": "^1.14.7",
+    "minimist": "^1.2.6"
```

Comment:
Hva brukes denne til?

Reply:
Det er en peer dependency av html-to-text, men ser ut som dne pakka ikke er i bruk lenger. @jeffreiffers ble lagt inn sammen med community categories page?